src/main/java/Kohli/Maincode/DeleteTask.java

```diff
@@ -0,0 +1,21 @@
+package Kohli.Maincode;
+
+import java.util.ArrayList;
+
+public class DeleteTask {
+    public static void removeTask(ArrayList<Task> tasks, int index) {
+        if (index < 0 || index >= tasks.size()) {
+            System.out.println("____________________________________________________________");
```

Comment:
Since you are using this line of code multiple times, you might want to save it as a variable or make it a function that you can reference multiple times.